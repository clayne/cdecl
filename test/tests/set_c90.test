cdecl @ @ @ set c90; set @ 0
