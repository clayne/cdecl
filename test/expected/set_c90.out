  noalt-tokens
  nodebug
  noeast-const
  noecho-commands
  noexplain-by-default
    explicit-ecsu=su
  noexplicit-int
  nographs
    lang=C89
    prompt
    semicolon
    using
